bitcoin/src/blockdata/transaction.rs

```diff
@@ -67,10 +67,9 @@ impl Txid {
 impl Wtxid {
     /// The `Wtxid` of a coinbase transaction.
     ///
-    /// This is used as the wTXID for the coinbase transaction when constructing blocks,
-    /// since the coinbase transaction contains a commitment to all transactions' wTXIDs
-    /// but naturally cannot commit to its own. It is not a real wTXID and should not be
-    /// used in other contexts.
+    /// This is used as the wTXID for the coinbase transaction when constructing blocks (in the
+    /// witness commitment tree) since the coinbase transaction contains a commitment to all
+    /// transactions' wTXIDs but naturally cannot commit to its own.
```

Comment:
I think we should keep "should not be used in other contexts" but probably also say "It does not identify a *unique* coinbase transaction".

Reply:
Added to #3331 to do as follow up.

Reply:
I was under the impression that this commit attempts to implement #3331 in which case I think it should do it well.

Reply:
That was not my intention, I'm trying not to queue PRs up in front of "move stuff to `primitives`" PRs, so follow ups come later.

Reply:
Iterating on docs is super trivial and need not hold up the main work.

Reply:
It is but I like to reduce the noise in git history whenever reasonable. It makes `git blame` less annoying for instance.

Reply:
Iterating for two weeks on every tiny piece of moving things to `primitives` is fucking killing me bro. I'm doing my best to maintain my sanity here but its hard.

Reply:
You could've skipped this entire commit if you don't want to iterate on documentation in these PRs. `primitives` doesn't need to be perfect from the start. The only thing I ask for is not to put non-required stuff into it *at first*, until we are 100% positive we want it there. (So that we don't have PRs doing the reverse of the work if we find out it's wrong.)